index.html

```diff
@@ -919,7 +919,10 @@ <h2 id="AudioContext">
             <p>
               Although the primary method of interfacing with this function is
               via its promise return value, the callback parameters are
-              provided for legacy reasons.
+              provided for legacy reasons. The system shall ensure that the
+              <code>AudioContext</code> is not garbage collected before the
```

Comment:
You can just surround keywords by an empty `<a>` tag, this makes a cross-reference.